maxim/models/evaluator.py

```
@@ -251,12 +302,23 @@ def dict_to_class(cls, data: Dict[str, Any]) -> "PassFailCriteria":
 
 @dataclass
 class LocalEvaluatorResultParameter:
+    """
+    This class represents the result parameter of a local evaluator.
+    """
+
     output: str
     context_to_evaluate: Optional[Union[str, List[str]]]
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)